src/PHONON/dynamical_matrix.cpp

```diff
@@ -87,12 +87,14 @@ void DynamicalMatrix::setup()
     neighbor->build(1);
 
     // compute all forces
-    external_force_clear = 0;
+    int ifix = modify->find_fix("package_omp");
```

Comment:
@charlessievers please note that as of PR #3001 the use of `Modify::find_fix()` is deprecated in favor of `Modify::get_fix_by_id()` (it will keep working for a while, but I have started to phase its use out). This makes the naming more consistent and saves one line as you can do:
```
if (!modify->get_fix_by_id("package_omp")) external_force_clear = 1;
```

Reply:
I will merge the develop branch and change those lines accordingly then, thanks for letting me know.